File: ce/customer-engagement/on-premises/snippets/visualbasic/CRMV8/queryvb/vb/createalinqquery1.vb
' <snippetcreatealinqquery1>


 Dim accounts = From a In svcContext.AccountSet _
                               Select New Account With _
                                      { _
                                          .Name = a.Name, _
                                          .Address1_County = a.Address1_County _
                                      }
 Console.WriteLine("List all accounts in CRM")
 Console.WriteLine("========================")
 For Each a In accounts
  Console.WriteLine(a.Name &amp; " " &amp; a.Address1_County)
 Next a

' </snippetcreatealinqquery1>

File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/query/cs/createalinqquery1.cs
// <snippetcreatealinqquery1>


 var accounts = from a in svcContext.AccountSet
                select new Account
                {
                 Name = a.Name,
                 Address1_County = a.Address1_County
                };
 System.Console.WriteLine("List all accounts in CRM");
 System.Console.WriteLine("========================");
 foreach (var a in accounts)
 {
  System.Console.WriteLine(a.Name + " " + a.Address1_County);
 }

// </snippetcreatealinqquery1>